### PR TITLE
Feature create game

### PR DESCRIPTION
**Pull request by arnarleifs**
Merging UI to develop branch

### CODE REVIEW

File: src/main/java/is/ru/tictactoe/BoardIndex.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    }
+
+    public int getRowIndex() {
+        return rowIndex;
+    }
+
+    public int getColIndex() {
+        return colIndex;
+    }
+}
```
Comment (hroar) [inline src/main/java/is/ru/tictactoe/BoardIndex.java]: Vantar newline í lokin

File: src/test/java/LibraryTest.java [hunk 2]
```diff
@@ -1,15 +0,0 @@
...
- * by 'baldur' at '10/20/15 10:44 AM' with Gradle 2.7
- *
- * @author baldur, @date 10/20/15 10:44 AM
- */
-public class LibraryTest {
-    @Test public void testSomeLibraryMethod() {
-        Library classUnderTest = new Library();
-        assertTrue("someLibraryMethod should return 'true'", classUnderTest.someLibraryMethod());
-    }
-}
```
Comment (BaldurTr) [inline src/test/java/LibraryTest.java]: Sad to see this one go!

Comment (arnarleifs) [inline src/test/java/LibraryTest.java]: I know :(

**Comment by bjoggib** [comment]
Too many merge conflicts.

**Pull request closed without merging**